pandas/core/indexing.py

```diff
@@ -443,6 +443,36 @@ def loc(self) -> _LocIndexer:
         8          4       5
         9          7       8
 
+        **Setting values using Series**
```

Comment:
i guess its ok to do this, but this is not what user's should be doing, so add a big warning that you you should simply be using `df['shield'] = ....` as the preferred syntax.

Reply:
@jreback Does the warning I've added now look sufficient? If not, how do you recommend changing it?

Reply:
actually this duplicates an example higher up. would prefer the warning in the Notes section (you can also add a comment before the example above)